### PR TITLE
W-10944696: Review GitHub documentation

### PR DESCRIPTION
**Pull request by mlischetti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,34 +76,38 @@ If the directory containing the `dw` executable is in your PATH, you can run `dw
...
  Documentation reference:
 
- https://docs.mulesoft.com/mule-runtime/4.3/dataweave
-    
-
+ https://docs.mulesoft.com/dataweave/latest/
 ```
 
-### Query content from a file
+### Querying the Content From a File
```
Comment (afelisatti) [inline README.md:108]:
```suggestion
### Querying Content From a File
```

File: README.md [hunk 2]
```diff
@@ -135,146 +133,73 @@ Input file `users.json`
...
 
-* *DW_HOME* = The directory where the home will be found if not defined `~/.dw` will be used
-* *DW_LIB_PATH* = The directory where libraries are going to be search by default. If not defined `${DW_HOME}/libs` will be used
-* *DW_DEFAULT_INPUT_MIMETYPE* = The default mimeType that is going to be used for the standard input. If not defined `application/json` will be used
-* *DW_DEFAULT_OUTPUT_MIMETYPE* = The default output mimeType that is going to be if not defined. If not defined `application/json` will be used
+```bash 
+dw "output application/xml --- users: {( 1 to 100 map (item) -> {user: "User" ++ item} )}" >> out.xml
+```
 
+### DataWeave CLI Environment Variables
```
Comment (afelisatti) [inline README.md:171]: Shouldn't we list this in a different section? Maybe under the help example? The other sections here look like a series of use cases.

File: README.md [hunk 3]
```diff
@@ -135,146 +133,73 @@ Input file `users.json`
...
+| `DW_DEFAULT_INPUT_MIMETYPE`  | The default `mimeType` that is going to be used for the standard input. If not defined `application/json` will be used. |
+| `DW_DEFAULT_OUTPUT_MIMETYPE` | The default output `mimeType` that is going to be if not defined. If not defined `application/json` will be used.       |
 
-An interesting use case for the CLI is to combine it with [curl](https://curl.haxx.se/)  
+## Power API Playground
 
+An interesting use case for the DataWeave CLI is to combine it with [curl](https://curl.haxx.se/)
 
-### Example query github commits
+### Query GitHub Commits
```
Comment (afelisatti) [inline README.md:183]: Thinking of the section below, maybe this should be less Github specific and just something like: `Query a GET Response`

File: README.md [hunk 4]
```diff
@@ -135,146 +133,73 @@ Input file `users.json`
...
 
-## CURL + DW => Power API Playground
+| Environment Variable         | Description                                                                                                             |
+|------------------------------|-------------------------------------------------------------------------------------------------------------------------|
+| `DW_HOME`                    | The directory where the home will be found if not defined `~/.dw` will be used.                                         |
+| `DW_DEFAULT_INPUT_MIMETYPE`  | The default `mimeType` that is going to be used for the standard input. If not defined `application/json` will be used. |
+| `DW_DEFAULT_OUTPUT_MIMETYPE` | The default output `mimeType` that is going to be if not defined. If not defined `application/json` will be used.       |
 
-An interesting use case for the CLI is to combine it with [curl](https://curl.haxx.se/)  
+## Power API Playground
```
[the remark this answers is not in the document]
Comment (mlischetti) [reply README.md]: I deleted the header by mistake. 
The original header was: "CURL + DataWeave => Power API Playground", that makes sense to you?

**Review by afelisatti — APPROVED** [review]

**Pull request merged**